app/src/main/java/com/codesoom/assignment/Method.java

```diff
@@ -0,0 +1,12 @@
+package com.codesoom.assignment;
+
+public enum Method {
```

Comment:
그냥 메소드는 너무 넓은 의미를 갖고 있습니다. 이렇게 해보시면 어떨까요?
```suggestion
public enum HttpMethod {
```

Reply:
수정해 보겠습니다. 적절한 이름 감사합니다~